<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PtRenderWidgetClass</class>
 <widget class="QDialog" name="PtRenderWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>点云渲染</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/coords.png</normaloff>img/coords.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radio_X">
       <property name="text">
        <string>X参数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_Y">
       <property name="text">
        <string>Y参数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_Z">
       <property name="text">
        <string>Z参数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_ok">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
